### PR TITLE
update standard to 0.6.0 and fix errors

### LINKED ISSUE #644: fix&test contact_made_chosen model validation (early return means error is never set)

### PR DESCRIPTION
**Pull request by jmkoni**
### What github issue is this PR for, if any?
Resolves #644 and updates standard to most recent version.

### What changed, and why?
Had to update standard because we are a few versions behind. Found a bug #644 and fixed that too.

### How will this affect user permissions?
Not at all.

### How is this tested? (please write tests!) 💖💪
Most was style issues, added a test for `contact_made_chosen`

### CODE REVIEW

File: app/decorators/case_contact_decorator.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def contact_types
   end
 
   def report_contact_types
-    object.contact_types&.map { |ct| ct.humanize.titleize }.join("|")
+    object.contact_types&.map { |ct| ct.humanize.titleize }&.join("|")
```
Comment (compwron) [inline app/decorators/case_contact_decorator.rb:50]: This should never be null in a saved case contact, I want to fix the underlying issue if this is causing issues

File: spec/models/case_contact_spec.rb [hunk 2]
```diff
@@ -65,4 +65,10 @@
     expect(case_contact).not_to be_valid
     expect(case_contact.errors[:base]).to eq(["Must enter miles driven to receive driving reimbursement."])
   end
+
+  it "validates that contact_made cannot be null" do
```
Comment (compwron) [inline spec/models/case_contact_spec.rb:69]: Yay test!

**Review by compwron — APPROVED** [review]
💖✨📈

File: app/decorators/case_contact_decorator.rb (hunk 1 above)
Comment (jmkoni) [reply app/decorators/case_contact_decorator.rb:50]: The issue was that we are doing the `&.map` but then calling join (on something that supposedly might not exist. Should we take both `&` out instead?

Comment (jmkoni) [reply app/decorators/case_contact_decorator.rb:50]: OMG you are so quick 😂

**Pull request merged**